src/Components/MapSyncRequestModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="MapSyncRequestModel.cs" company="Talegen">
```

Comment:
GlobalLink Vasont

Reply:
what the heck? didn't notice that. It's how they are in Framework